adaptors/ampersand/base_view.js

```diff
@@ -31,10 +31,6 @@ var BaseView = AmpersandView.extend({
     }
     this.options = options || {};
 
-    // VTree is passable as an option if we are transitioning in from a different view
```

Comment:
If you look at that file, that if block was in there twice:
https://github.com/wayfair/tungstenjs/blob/master/adaptors/ampersand/base_view.js#L35
https://github.com/wayfair/tungstenjs/blob/master/adaptors/ampersand/base_view.js#L42